generators/server/templates/pom.xml.ejs

```diff
@@ -1189,6 +1189,11 @@
                             <artifactId>spring-beans</artifactId>
                             <version>${spring.version}</version>
                         </dependency>
+                        <dependency>
```

Comment:
it should not be present for NoSQL database as there is no liquibase

Reply:
@Tcharl : we don't have Liquibase for MongoDB, Couchbase, Cassandra, so I still think adding this for these database is useless